cmd/edit/machinepool/cmd.go

```diff
@@ -245,8 +273,76 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	labels := args.labels
+	labelMap := make(map[string]string)
+	if interactive.Enabled() {
+		if labels == "" {
+			for lk, lv := range machinePool.Labels() {
+				if labels != "" {
+					labels += ","
+				}
+				labels += fmt.Sprintf("%s=%s", lk, lv)
+			}
+		}
+		labels, err = interactive.GetString(interactive.Input{
+			Question: "Labels",
+			Help:     cmd.Flags().Lookup("labels").Usage,
+			Default:  labels,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid comma-separated list of attributes: %s", err)
+			os.Exit(1)
+		}
+	}
+	labels = strings.Trim(labels, " ")
+	if labels != "" {
```

Comment:
How can the user remove the existing labels? not by sending `""` ?
You might want to allow `""` as input and change this chek to `Changed("labels")` bool